cypress/support/commands.js

```diff
@@ -23,14 +23,37 @@ Cypress.Commands.add('deviceApproval', () => {
     cy.get('#nav-toggle').click()
     cy.contains('Edge Management').click()
     cy.contains('Devices').click()
-    cy.get('[data-label="Approve"]').should('exist')
-    cy.get('[data-label="Approve"]').should('be.visible')
-    cy.get('[data-label="Approve"]').click()
+    cy.get(':nth-child(1) > .pf-v5-l-flex > :nth-child(1) > .pf-v5-c-title').contains('Devices pending approval')
+    cy.get('[data-label="Approve"] > .pf-v5-c-button').should('exist')
+    cy.get('[data-label="Approve"] > .pf-v5-c-button').should('be.visible')
+    cy.get('[data-label="Approve"] > .pf-v5-c-button').click()
     cy.get('#rich-validation-field-deviceAlias').should('be.visible')
     cy.get('#rich-validation-field-deviceAlias').type('test-device')
     cy.get('#rich-validation-field-deviceAlias').should('have.value', 'test-device')
     cy.get('.pf-v5-c-form__actions > .pf-m-primary').should('be.visible')
     cy.get('.pf-v5-c-form__actions > .pf-m-primary').click()
+    cy.get('[data-label="Device status"]', { timeout: 50000 }).should('contain', 'Online')
+})
+Cypress.Commands.add('editDevice', (img=`${Cypress.env('image')}`) => {
+    cy.get('#nav-toggle').should('exist')
+    cy.get('#nav-toggle').click()
+    cy.contains('Edge Management').click()
+    cy.contains('Devices').click()
+    cy.get('a > .fctl-resource-link__text').contains('test-device')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').click()
+    cy.contains('Edit device configurations').click()
+    cy.get('#rich-validation-field-deviceAlias').should('be.visible')
+    cy.get('#rich-validation-field-deviceAlias').should('have.value', 'test-device')
+    cy.get('#rich-validation-field-deviceAlias').clear()
+    cy.get('#rich-validation-field-deviceAlias').type('test-device-edited')
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('#textfield-osImage').should('be.visible')
+    cy.get('#textfield-osImage').type(img)
+    cy.get('#textfield-osImage').should('have.value', 'quay.io/sdelacru/flightctl-centos:v1')
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
```

Comment:
Just my opinion, but I really prefer defining page object / access methods for navigating the elements in the page.

- It makes the code explicit about what the intent is, and that makes the code easy to read and understand, specially when some random test fails.
- It prevents repetition so that if the selector changes, a single function needs to be updated vs every ocurrence of the selector.



```suggestion
    wizardFooter.getNextButton().click()
```